src/formatDate.js

```diff
@@ -50,7 +50,46 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const SPLITTER_IN = fromFormat[3];
+  const SPLITTER_OUT = toFormat[3];
+  const compDateArrIn = date.split(SPLITTER_IN);
+  const arrOut = [];
+
+  let yearStrIn2;
+  let yearStrIn4;
+  const monthStrIn = compDateArrIn[fromFormat.indexOf('MM')];
+  const dayStrIn = compDateArrIn[fromFormat.indexOf('DD')];
+
+  if (fromFormat.includes('YY')) {
```

Comment:
You can create a variable for all magic words and use them everywhere
```suggestion
  if (fromFormat.includes('YY')) {
```